basiclti/basiclti-impl/src/java/org/sakaiproject/lti/impl/LTIExportServiceImpl.java

```diff
@@ -0,0 +1,254 @@
+package org.sakaiproject.lti.impl;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.OutputStream;
+import java.text.DateFormat;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.apache.poi.hssf.usermodel.*;
+import org.apache.poi.ss.util.WorkbookUtil;
+import org.sakaiproject.lti.api.LTIExportService;
+import org.sakaiproject.lti.api.LTIService;
+import org.sakaiproject.site.api.SiteService;
+import org.sakaiproject.component.api.ServerConfigurationService;
+import org.sakaiproject.util.FormattedText;
+import org.sakaiproject.util.ResourceLoader;
+
+public class LTIExportServiceImpl implements LTIExportService {
+	protected LTIService ltiService;
+	protected ServerConfigurationService serverConfigurationService;
+	protected SiteService siteService;
+	
+	private String csvSeparator = null;
+	
+	/** Resource bundle using current language locale */
+	protected static ResourceLoader rb = new ResourceLoader("ltiservice");
+	
+	private static Log M_log = LogFactory.getLog(LTIExportServiceImpl.class);
+	
+	public void setLtiService(LTIService ltiService) {
+		this.ltiService = ltiService;
+	}
+	
+	public void setServerConfigurationService(ServerConfigurationService serverConfigurationService) {
+		this.serverConfigurationService = serverConfigurationService;
+	}
+
+	public void setSiteService(SiteService siteService) {
+		this.siteService = siteService;
+	}
+
+	public void init() {
+		
+	}
+	
+	private boolean isAdmin(String siteId) {
+		if ( siteId == null ) {
```

Comment:
Because is not checking for super user permissions just checks if has maintain access. This is copied from BaseLTIService, but maybe we should change the name for that method.